examples/web-wagmi/package.json

```diff
@@ -26,19 +26,20 @@
     "react-hot-toast": "^2.4.0",
     "react-router-dom": "^6.8.1",
     "readable-web-to-node-stream": "^3.0.2",
-    "wagmi": "0.9.2"
+    "wagmi": "^0.11.6"
```

Comment:
I think we need to be careful with wagmi. It's in initial development phase (`0.x`) and they definitely make breaking changes, we experiences one ourself between `0.8.x` and `0.9.x`. Given there is no code changes required on our side between `0.9.2` and `0.11.6` I would consider an range: `">=0.9.2 <=0.11.6"`.

@desfero what do you think?

Reply:
agree, we need to narrow the `wagmi` to a specific range that we know works properly inside `@lens-sdk/wagmi` `peerDependencies` configuration.
We probably needs some changeset that's mentioning which important deps were updated